var/spack/repos/builtin/packages/py-apipkg/package.py

```diff
@@ -12,6 +12,7 @@ class PyApipkg(PythonPackage):
     homepage = "https://pypi.python.org/pypi/apipkg"
     url      = "https://pypi.io/packages/source/a/apipkg/apipkg-1.4.tar.gz"
 
-    version('1.4', '17e5668601a2322aff41548cb957e7c8')
+    version('1.5', sha256='2e38399dbe842891fe85392601aab8f40a8f4cc5a9053c326de35a1cc0297ac6')
+    version('1.4', sha256='37228cda29411948b422fae072f57e31d3396d2ee1c9783775980ee9c9990af6')
 
```

Comment:
I would also add the following dependency:
```python
depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
```

Reply:
Thanks!